website/docs/r/cos_bucket.html.markdown

```diff
@@ -158,16 +178,24 @@ The following arguments are supported:
 * `endpoint_type` - (Optional, string) The type of the endpoint (public or private) to be used for buckets. Default value is `public`.
 
 * `force_delete` - (Optional, bool) Since Default value set to True, it will delete all the objects in the COS Bucket and then delete the bucket.  Default value is `true`.
+    * **Note** - `force_delete` will timeout on buckets with a large amount of objects.  24 hours before you delete the bucket you can set an expire rule to remove all files over a day old.  
 
 * Nested `archive_rule` block have the following structure:
 	*	`archive_rule.rule_id` : (Optional, Computed, string) Unique identifier for the rule. Archive rules allow you to set a specific time frame after which objects transition to the archive. 
 	*	`archive_rule.enable` :* (Required, bool) (Required) Specifies archive rule status either enable or disable for a bucket.
 
-  *	`archive_rule.days` :* (Required, string)  Specifies the number of days when the specific rule action takes effect.
+    *	`archive_rule.days` :* (Required, string)  Specifies the number of days when the specific rule action takes effect.
```

Comment:
I am not getting the `*` after the `:` on several of these fields. It is not consistent (like not all required ones it is on one optional one below)...